show/reboot_cause.py

```diff
@@ -30,7 +30,10 @@ def reboot_cause(ctx):
         # Read the previous reboot cause
         data = read_reboot_cause_file()
         if data['user'] == "N/A":
-            reboot_cause = "{}".format(data['cause'])
+            if data['cause'] == "Kernel Panic":
+                reboot_cause = "{} (Time: {})".format(data['cause'], data['time'])
+            else: 
+                reboot_cause = "{}".format(data['cause'])
         else:
             reboot_cause = USER_ISSUED_REBOOT_CAUSE_REGEX.format(data['cause'], data['user'], data['time'])
```

Comment:
Maybe we can make this more generic such that it appends user and/or time to the output if they are not 'N/A'?

Reply:
Agree and great suggestion. Will update.